server/inspect_test.go

```diff
@@ -1,217 +1,130 @@
-		}
-	}
-	for k, v := range labels {
-		found = false
-		for key, value := range ci.Labels {
-			if k == key && v == value {
-				found = true
-				break
-			}
-		}
-		if !found {
-			t.Fatalf("key %s with value %v wasn't in container info labels", k, v)
-		}
-	}
-}
-
-func TestGetContainerInfoCtrNotFound(t *testing.T) {
-	s := &Server{}
-	getContainerFunc := func(id string) *oci.Container {
-		return nil
-	}
-	getInfraContainerFunc := func(id string) *oci.Container {
-		return nil
-	}
-	getSandboxFunc := func(id string) *sandbox.Sandbox {
-		return nil
-	}
-	_, err := s.getContainerInfo("", getContainerFunc, getInfraContainerFunc, getSandboxFunc)
-	if err == nil {
-		t.Fatal("expected an error but got nothing")
-	}
-	if err != errCtrNotFound {
-		t.Fatalf("expected errCtrNotFound error, got %v", err)
-	}
-}
-
-func TestGetContainerInfoCtrStateNil(t *testing.T) {
-	s := &Server{}
-	created := time.Now()
-	labels := map[string]string{}
-	annotations := map[string]string{}
-	getContainerFunc := func(id string) *oci.Container {
-		container, err := oci.NewContainer("testid", "testname", "", "/container/logs", "", labels, annotations, annotations, "imageName", "imageName", "imageRef", &runtime.ContainerMetadata{}, "testsandboxid", false, false, false, false, false, "", "/root/for/container", created, "SIGKILL")
-		if err != nil {
-			t.Fatal(err)
-		}
-		container.SetMountPoint("/var/foo/container")
-		container.SetState(nil)
-		return container
-	}
-	getInfraContainerFunc := func(id string) *oci.Container {
-		return nil
-	}
-	getSandboxFunc := func(id string) *sandbox.Sandbox {
-		s := &sandbox.Sandbox{}
-		s.AddIP("1.1.1.42")
-		return s
-	}
-	_, err := s.getContainerInfo("", getContainerFunc, getInfraContainerFunc, getSandboxFunc)
-	if err == nil {
-		t.Fatal("expected an error but got nothing")
-	}
-	if err != errCtrStateNil {
-		t.Fatalf("expected errCtrStateNil error, got %v", err)
-	}
-}
-
-func TestGetContainerInfoSandboxNotFound(t *testing.T) {
-	s := &Server{}
-	created := time.Now()
-	labels := map[string]string{}
-	annotations := map[string]string{}
-	getContainerFunc := func(id string) *oci.Container {
-		container, err := oci.NewContainer("testid", "testname", "", "/container/logs", "", labels, annotations, annotations, "imageName", "imageName", "imageRef", &runtime.ContainerMetadata{}, "testsandboxid", false, false, false, false, false, "", "/root/for/container", created, "SIGKILL")
-		if err != nil {
-			t.Fatal(err)
-		}
-		container.SetMountPoint("/var/foo/container")
-		return container
-	}
-	getInfraContainerFunc := func(id string) *oci.Container {
-		return nil
-	}
-	getSandboxFunc := func(id string) *sandbox.Sandbox {
-		return nil
-	}
-	_, err := s.getContainerInfo("", getContainerFunc, getInfraContainerFunc, getSandboxFunc)
-	if err == nil {
-		t.Fatal("expected an error but got nothing")
-	}
-	if err != errSandboxNotFound {
-		t.Fatalf("expected errSandboxNotFound error, got %v", err)
-	}
-}
+// The actual test suite
```

Comment:
nit: did you mean to leave this?

Reply:
Hm, I guess this comment can be removed.